brilift/src/main.rs

```diff
@@ -61,13 +113,13 @@ impl RTFunc {
 
 /// Runtime functions used in the native `main` function, which dispatches to the proper Bril
 /// `main` function.
-#[derive(Debug, Enum)]
+#[derive(Debug)]
 enum RTSetupFunc {
     ParseInt,
     ParseBool,
 }
 
-impl RTSetupFunc {
+impl RTFuncUtils for RTSetupFunc {
```

Comment:
Sharing infrastructure between the two categories of runtime functions may also be a good idea. The current separation conflates two things:

* some functions are only used in the C main wrapper
* those functions also happen to be the only ones whose signatures care what size pointers are on the target platform

Those are really not essentially conflated. It's possible that the best design would just combine them all into one enum…